### ISSUE TITLE
Don't purge 'output DB' for each 'input scenario/tool' in merger/importer

### ISSUE THREAD

**Issue by manuelma** [Spine-project/Spine-Toolbox#1768]
It may happen that Merger is executing multiple scenarios or tools in parallel - at the moment the output DBs will be purged independently for each such execution.

So each filtered merger execution would potentially undo the work of the other executions.

To prevent this, we need to do the purging only once. For consistency Importer should do the same but at the moment Importer can't be running multiple filtered executions?

**Comment by jkiviluo** [comment]
Good catch. When I run the importer with multiple scenarios selected, they do execute in parallel. And with purge active, I only get one scenario to the DB.

Another issue is that often the desired outcome is to purge only the scenario(s) you are running. Would it work if we had 'If scenario already exists: [ ] keep existing, [ ] replace the latest'?

**Comment by manuelma** [comment]
I think this works properly now. Will close, please reopen if needed.

**Comment by soininen** [comment]
This should fix #1697 as well. Nice work!
